Buildings/Experimental/DHC/Loads/Examples/BaseClasses/BuildingTimeSeries.mo

```diff
@@ -2,19 +2,21 @@ within Buildings.Experimental.DHC.Loads.Examples.BaseClasses;
 model BuildingTimeSeries
   "Building model with heating and cooling loads provided as time series"
   extends Buildings.Experimental.DHC.Loads.BaseClasses.PartialBuilding(
-    redeclare package Medium=Buildings.Media.Water,
+    redeclare package Medium=MediumW,
     final have_fan=false,
     final have_pum=true,
     final have_eleHea=false,
     final have_eleCoo=false,
     final have_weaBus=false);
-  package Medium2=Buildings.Media.Air
-    "Load side medium";
+  package MediumA=Buildings.Media.Air
+    "Air medium";
+  package MediumW=Buildings.Media.Water
+    "Water medium";
   parameter String filNam
     "File name with thermal loads as time series";
-  parameter Real facScaHea=10
+  parameter Real facScaHea=1
     "Heating terminal unit scaling factor";
```

Comment:
The comment of the scaling factor is not clear to me. It seems like that the original scaling factor is to break down the load to individual terminal unit. This is why I see originally in the modification of `QCoo_flow_nominal`, `min = Buildings.Experimental.DHC.Loads.BaseClasses.getPeakLoad()` without multuplying`facScaCoo`.
But the new change intends to use the scaling factor to scale up from one terminal unit to whole building?

Reply:
the intention of the change is to avoid the trial and error approach used previously to customize the scaFacHea, scaFacCoo for the implemented time series loads file. If the user change the time series loads file, he/she will face an error like that

![image](https://user-images.githubusercontent.com/47648835/101533589-40f81100-3953-11eb-9e59-e504eb5c96c4.png)
 i.e. error indicates a violation  of the 2nd law of thermodynamics at the terminal unit HX. 

In addition , Only one heating terminal unit is serving the building heating loads and only one cooling terminal unit is serving the buildings cooling loads, we dont have buildings zones for multiple units. After the current changes if the user "for some reason" wants to scale up or down the QHea/QCoo_nominal  i.e. design parameters of the heating / cooling terminal unit the model will still run without any error.. If not, {1} shall be the default value for the two parameters.

Reply:
**The rational behind the changes is to make the model work at the GMT because currently it fails if the time series loads file changes.** 

The changes are to avoid using the trail and error approach used at the current model to evaluate two customized scaling factor parameters every time the time series file change because if we don't do that, the model will simply fail.

**Discussion**
We consider the time series loads model as a special type of UO building loads since we don't have thermal zones, AHU data , FCU data , whether a radiant floors are implemented or not etc,  all what we know is the aggregated time series thermal loads for the whole building's zones.

So we assume to have one ideal terminal unit to handle the heating time series loads and one ideal terminal unit to handle the cooling time series loads. Hence I believe that for this specific type of building loads, we have to consider “mentally” a larger capacity systems, and not only hover around the limited capacity of a FCU.

On the other hand, the commercial FCU nominal air flow rate could start from 300cfm to 3000cfm, hence there is no universal nominal air flow = 1kg/s that we should commit to, i mean what if the user decided to change the nominal flow rate to 0.5 kg/s, this change will need at least two weeks of trial and error  efforts “as per your experience in evaluating the 10 and 40 values” to find the correct combination of <code>scaFacHea and scaFacCoo </code>.

For the sake of brevity, depending on "trial and error evaluated parameters" is not an extensible approach for the GMT.

**Conclusion**
The <code>scaFac</code>10  and  40 both are evaluated arbitrarily i.e by trial and error, and if the time series file changes; the model will fail and we are trying to solve these problems because it will provoke the extensibility of the UrbanOpt_GeoJson Modelica Translator UO_GMT.